src/pages/Committees.js

```diff
@@ -580,11 +634,213 @@ function SportsCommittee() {
+            The Study Board (Swedish: Studienämnden) verifies that every student has a well-functioning education and safe
+            study environment. This applies at both programme-level and course-level. If you have any problem with courses and
+            programs, or have questions about them, you can contact the Study Board! Joining the committee lets you help everyone
+            have a better education and you will have a better understanding of education at KTH.
+            <br/>
+            Contact:
+            <ul>
+                <li>Email: sno@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function KomN () {
+    return (
+        <p>
+            The Communication Committee (Swedish: Kommunikationsnämnden) is the committee in charge of Media's external branding,
+            e.g. they update Media's Instagram, Facebook, etc. They also propose fun, short cohesion events for the chapter and run a
+            podcast on Spotify! In addition to this, they propose an advent calender made of 24 short videos.
+            <br/>
+            Contact:
+            <ul>
+                <li>Email: kommunikator@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function MBD () {
+    return (
+        <p>
+            Searching for a job after KTH? Media's carrier fair (Swedish: Medias Branschdag) is a large event that lets you find jobs,
+            seek degree project opportunities or simply enrich your network, it's up to you! Don't miss it!
+            <br/>
+            Contact:
+            <ul>
+                <li>Email: branschdag@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function MediaNLG () {
+    return (
+        <p>
+            The Business Relations Group (Swedish: Näringslivsgruppen) initiate contacts with companies and provide students with opportunities
+            to build a strong network of business contacts. Working for the Business Group makes it doubly as effective for you. Members
+            organise events such as mingles or recruitement events.
+            <br/>
+            Contact:
+            <ul>
+                <li>Email: nlg@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function Festmästeriet () {
+    return (
+        <p>
+            Party people, here is your stuff! The Party Committee (Swedish: Festmästeriet) is specialized in organizing and hosting gasques;
+            traditional swedish student parties featuring a three course meal, fun activities and an afterparty open for everyone.
+            Members of the committee can engage in various tasks, such as deciding the menu, doing PR (i.e. marketing), inventing fun
+            competitions, or decorating the venue. Join them to maximise the fun of Media's parties.
+            <br/>
+            Contact:
+            <ul>
+                <li>Email: festmasteriet@medieteknik.com</li>
+                <li>Instagram: @festmasteriet_medieteknik</li>
```

Comment:
```suggestion
                <li><a href="https://www.instagram.com/festmasteriet_medieteknik/">Instagram</a></li>
```

Reply:
I can't access that link. It gives me the following error (translated from French):
> An error happened.
> An error happened and it is impossible to charge the page.
There is a reload button, but it doesn't do anything.

Reply:
That's wierd, it works for me. Are you sure you copied and pasted it correctly? [https://www.instagram.com/festmasteriet_medieteknik/](https://www.instagram.com/festmasteriet_medieteknik/)
If it still doesn't work, you can just remove the link and leave it as it was before.

Reply:
I don't know why this happens (I re-tried with the link you sent here). It's not the only Instagram page that doesn't work for me. I can add the link if others can see it.

Reply:
I don't get why, now I don't have the error anymore. I'll update this.